test/v2/ThirdPartyRegistry.spec.js

```diff
@@ -4544,6 +4549,468 @@ describe('ThirdPartyRegistry', function () {
     })
   })
 
+  describe('reviewThirdPartyWithRoot', function () {
+    it('should update the third party and emit the reviewed event when consume slots is empty', async function () {
```

Comment:
I may change the description as

`should review a third party :: no slots consumption`
`should review a third party :: with slots consumption :: one signature`
`should review a third party :: with slots consumption :: multiple signatures`

Reply:
Added a new test for `should review a third party :: with slots consumption :: one signature` as there was none for just 1 case